tools/eslint/package.json

```diff
@@ -14,19 +14,19 @@
     ]
   ],
   "_from": "eslint@latest",
-  "_id": "eslint@3.19.0",
+  "_id": "eslint@4.0.0",
```

Comment:
(Do you mean that it might be better to build from source or something rather than install via npm?)